.github/workflows/python.yml

```
@@ -36,33 +37,22 @@ jobs:
           git config user.email "<>"
           git --version
 
-      - name: Package Python 3.8 and earlier
+      - name: Package Python 3.8 and earlier for Linux x86_64
         if: ${{ startsWith(matrix.os, 'ubuntu') && matrix.pyver == '3.8' }}
         run: docker run --rm -v ${{github.workspace}}/:/io ezralanglois/interop sh /io/tools/package.sh /io /io/dist travis OFF
 
-      - name: Package Python 3.9
-        if: ${{ startsWith(matrix.os, 'ubuntu') && matrix.pyver == '3.9' }}
-        run: docker run --rm -v ${{github.workspace}}/:/io $DOCKER_IMAGE sh /io/tools/package.sh /io /io/dist travis OFF Release cp39-cp39
+      - name: Package Python 3.9-3.x Linux x86_64
+        if: ${{ startsWith(matrix.os, 'ubuntu') }}
+        run: docker run --rm -v ${{github.workspace}}/:/io $DOCKER_IMAGE sh /io/tools/package.sh /io /io/dist travis OFF Release ${{ matrix.pyver }}
 
-      - name: Package Python 3.10
-        if: ${{ startsWith(matrix.os, 'ubuntu') && matrix.pyver == '3.10' }}
-        run: docker run --rm -v ${{github.workspace}}/:/io $DOCKER_IMAGE sh /io/tools/package.sh /io /io/dist travis OFF Release cp310-cp310
-
-      - name: Package Python 3.11
-        if: ${{ startsWith(matrix.os, 'ubuntu') && matrix.pyver == '3.11' }}
-        run: docker run --rm -v ${{github.workspace}}/:/io $DOCKER_IMAGE sh /io/tools/package.sh /io /io/dist travis OFF Release cp311-cp311
-
-      - name: Package Python 3.12
-        if: ${{ startsWith(matrix.os, 'ubuntu') &&  matrix.pyver == '3.12' }}
-        run: docker run --rm -v ${{github.workspace}}/:/io $DOCKER_IMAGE sh /io/tools/package.sh /io /io/dist travis OFF Release cp312-cp312
-
-      - name: Package Python 3.13
-        if: ${{ startsWith(matrix.os, 'ubuntu') &&  matrix.pyver == '3.13' }}
-        run: docker run --rm -v ${{github.workspace}}/:/io $DOCKER_IMAGE sh /io/tools/package.sh /io /io/dist travis OFF Release cp313-cp313
+      - name: Install QEMU
+        if: ${{ startsWith(matrix.os, 'ubuntu') }}
+        run: |
+          docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
 
-      - name: Package Python 3.14
-        if: ${{ startsWith(matrix.os, 'ubuntu') &&  matrix.pyver == '3.14' }}
-        run: docker run --rm -v ${{github.workspace}}/:/io $DOCKER_IMAGE sh /io/tools/package.sh /io /io/dist travis OFF Release cp314-cp314
+      - name: Package Python 3.9-3.x Linux Arm64
+        if: ${{ startsWith(matrix.os, 'ubuntu') }}
+        run: docker run --rm -v ${{github.workspace}}/:/io $DOCKER_IMAGE_AARCH64 sh /io/tools/package.sh /io /io/dist travis OFF Release ${{ matrix.pyver }}
```

Even though you're setting up QEMU above, the `$DOCKER_IMAGE_AARCH64` is built with ARM64 as target platform. You'll see errors in the build logs in the form of:
```
WARNING: The requested image's platform (linux/arm64) does not match the detected host platform (linux/amd64/v3) and no specific platform was requested
```
This ***could*** be the reason for some of the compilation errors you're encountering and trying to fix in this PR. Either you configure the GitHub workflow to use ARM64 runner for this step (more convoluted), or tell Docker the target architecture of the image you want to run:
```suggestion
        run: docker run --rm --platform linux/arm64 -v ${{github.workspace}}/:/io $DOCKER_IMAGE_AARCH64 sh /io/tools/package.sh /io /io/dist travis OFF Release ${{ matrix.pyver }}
```

---

It is worth testing

---

It does not seem to help with the segfaults